Examples/installation_verification.py

```diff
@@ -1,27 +1,84 @@
 from XRPLib.defaults import *
-from .drive_examples import *
-from .misc_examples import *
 import time
 
 # Installation Verification Program
 def ivp():
+    # Print welcome message
+    print("------------------------------------------")
+    print("Running Installation Verification Program!")
+    print("------------------------------------------")
+    print()
+    print("Welcome to the XRP! This example code will help")
+    print("you verify that everything is working on your robot.")
+    print("After each test, press the user button to continue.")
+    print()
+    
+    # Flash LED at 5Hz
+    print("Flashing LED")
+    board.led_blink(5)
+    
+    # Wait for user to press button
+    print("Press user button to test reflectance sensor")
+    board.wait_for_button()
+    
+    # Stop blinking LED
+    board.led_off()
+    
+    # Print reflectance values until button is pressed
     while not board.is_button_pressed():
-        print(f"Left Reflectance: {reflectance.get_left()}, Right Reflectance: {reflectance.get_right()}")
+        print(f"Reflectance    Left: {reflectance.get_left():.3f}    Right: {reflectance.get_right():.3f}    Press user button to test range sensor")
         time.sleep(0.1)
+    
+    # Wait until button is released
     while board.is_button_pressed():
-        time.sleep(.01)
+        time.sleep(0.1)
+    
+    # Print range values until button is pressed
     while not board.is_button_pressed():
-        print(f"Ultrasonic Distance: {rangefinder.distance()}")
+        print(f"Range    Distance: {rangefinder.distance():.1f}    Press user button to test servo")
         time.sleep(0.1)
+    
+    # Wait until button is released
     while board.is_button_pressed():
-        time.sleep(.01)
-    print("Testing Servo")
-    test_servo()
-    print("Testing LEDs")
+        time.sleep(0.1)
+    
+    # Print warning and wait for button press
+    print()
+    print("The next test will move the servo, so keep your hands clear!")
+    print("Also please make sure the power switch is turned on!")
+    print("Press user button to test servo")
     board.wait_for_button()
-    test_leds()
-    print("Testing Drivetrain:")
+    
+    # Test servo
+    print("Testing servo")
+    time.sleep(1)
+    servo_one.set_angle(90)
+    time.sleep(1)
+    servo_one.set_angle(0)
+    time.sleep(1)
+    servo_one.set_angle(180)
+    
+    # Print warning and wait for button press
+    print()
+    print("The next test will drive the motors, so place the robot on a flat surface!")
+    print("Also please make sure the power switch is turned on!")
+    print("Press user button to test drivetrain")
     board.wait_for_button()
-    test_drive()
+    
+    # Test drivetrain
+    print("Testing drivetrain")
+    time.sleep(1)
+    drivetrain.straight(25, 0.8)
+    time.sleep(1)
+    drivetrain.turn(90,0.8)
```

Comment:
calling turn implicitly tests the imu, but do we want to test it explicitly?

Reply:
I don't have a strong preference either way, as long as it gets tested. It's also the same with the encoders. I would lean towards implicit testing to reduce the amount of code, and reduce time spent testing by the user. But again, no strong preference.

Reply:
Yeah, i'm inclined to agree, as long as people are actually testing that the bot is moving 25cm and turning 90 degrees, then we're fine. gonna approve PR now since everything is good